src/tools/dev/scripts/yearly_commits.py

```diff
@@ -23,26 +23,26 @@ def increment_for_version(ver):
 def main(startver, endver):
     hits_per_month = {}
     count = 0
-    for v in xrange(startver, endver):
+    for v in range(startver, endver):
```

Comment:
I think this script is pretty much useuless now that we've abandon subversion

Reply:
we do need to clean lots  up scripts

Reply:
added #4193